src/sentry_value.c

```diff
@@ -461,6 +461,38 @@ sentry_value_append(sentry_value_t value, sentry_value_t v)
     return 0;
 }
 
+sentry_uuid_t
+sentry__value_as_uuid(sentry_value_t value)
+{
+    const char *val = sentry_value_as_string(value);
+    if (val) {
+        return sentry_uuid_from_string(val);
+    } else {
+        return sentry_uuid_nil();
+    }
+}
+
+char *
+sentry__value_stringify(sentry_value_t value)
+{
+    switch (sentry_value_get_type(value)) {
+    case SENTRY_VALUE_TYPE_LIST:
+    case SENTRY_VALUE_TYPE_OBJECT:
+    case SENTRY_VALUE_TYPE_NULL:
+        return sentry__string_clone("");
+    case SENTRY_VALUE_TYPE_BOOL:
+        return sentry__string_clone(
+            sentry_value_is_true(value) ? "true" : "false");
+    case SENTRY_VALUE_TYPE_STRING:
+        return sentry__string_clone(sentry_value_as_string(value));
+    default: {
+        char buf[50];
+        snprintf(buf, sizeof(buf), "%g", sentry_value_as_double(value));
```

Comment:
Will this create precision issues with large numbers? It might be better to explicitly handle `INT32`.

Reply:
shouldn’t `f64` be able to exactly represent any `u32`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

Reply:
You. are. right. Sir.